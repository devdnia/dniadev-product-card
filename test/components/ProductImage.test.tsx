import React from 'react';
import renderer from 'react-test-renderer';
import { ProductImage, ProductCard } from '../../src/components';
import { product2 } from './data/products';



describe('ProductImage', () =>{

    test('Debe mostrar la URL de la image', () =>{

        const wrapper = renderer.create(
            <ProductImage img='url/imagen' className='class-image'/>,
        )
        expect( wrapper.toJSON()).toMatchSnapshot();
    }),

    test('Debe de mostrar el compomente con la URL de la imagen', () => {

        const wrapper = renderer.create(
            <ProductCard product={ product2 } >
                {
                    () => (
                        <ProductImage img={ product2.img }/>
                    )
                }
            </ProductCard>
        )
        expect( wrapper.toJSON()).toMatchSnapshot();

    })

})