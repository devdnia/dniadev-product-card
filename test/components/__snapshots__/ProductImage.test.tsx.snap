// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage Debe de mostrar el compomente con la URL de la imagen 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product image"
    className="productImg undefined"
    src="./coffÂ§ee-mug.png"
  />
</div>
`;

exports[`ProductImage Debe mostrar la URL de la image 1`] = `
<img
  alt="Product image"
  className="productImg class-image"
  src="url/imagen"
/>
`;
